Formula/h/halide.rb

```diff
@@ -29,40 +28,27 @@ class Halide < Formula
   depends_on "libpng"
   depends_on "lld"
   depends_on "llvm"
+  depends_on "openssl@3"
```

Comment:
looks like this is macos only dep https://github.com/Homebrew/homebrew-core/actions/runs/12387624970/job/34579673357?pr=201454#step:5:184